oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxNetworkIF.java

```diff
@@ -211,7 +219,29 @@ public boolean updateAttributes() {
         // speed may be -1 from file.
         this.speed = speedMiB < 0 ? 0 : speedMiB << 20;
         this.ifAlias = FileUtil.getStringFromFile(ifAliasPath);
+        this.ifOperStatus = mapIfOperStatus(FileUtil.getStringFromFile(ifOperStatusPath));
 
         return true;
     }
+
+    private IfOperStatus mapIfOperStatus(String operState) {
```

Comment:
(Preference) I'd avoid using "map" unless the return value is a `Map<>`.  I think `parse...` may be better?